namelayer-spigot/pom.xml

```diff
@@ -25,21 +25,15 @@
     <dependency>
       <groupId>org.spigotmc</groupId>
       <artifactId>spigot-api</artifactId>
-      <version>1.12.2-R0.1-SNAPSHOT</version>
+      <version>1.12-R0.1-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
 
     <!-- Civcraft -->
-    <dependency>
-      <groupId>vg.civcraft.mc.mercury</groupId>
-      <artifactId>Mercury</artifactId>
-      <version>1.2.19</version>
-      <scope>provided</scope>
-    </dependency>
     <dependency>
       <groupId>vg.civcraft.mc.civmodcore</groupId>
       <artifactId>CivModCore</artifactId>
-      <version>1.6.0</version>
+      <version>1.6.2</version>
```

Comment:
I have it on my todo, but 1.6.2 is a custom CivModCore release specific to CivClassic, and is not yet hosted by Devoted's buildstack. I've checked out the commits that consist of 1.6.2 and have no objection, however, I have to do it in a branch as master on Devoted has already moved to 1.7.x for MC 1.13.x support.

Reply:
So i guess we'll look at that first then

Reply:
Actually this can just be 1.6.1, the 1.6.2 changes are entirely unrelated